### PR TITLE
Add `normalize` option to `gausshermite`

### PR DESCRIPTION
**Pull request by NittanyLion**
I've added an option that allows users to specify that they want the probabilist version of the Gauss Hermite weights.  The documentation has been updated, also.

### CODE REVIEW

**Review by hyrodium — COMMENTED** [review]
Could you add some tests for this change?

**Comment by NittanyLion** [comment]
> Could you add some tests for this change?

sure, thanks

File: src/gausshermite.jl [hunk 1]
```diff
@@ -1,12 +1,18 @@
 @doc raw"""
-    gausshermite(n::Integer) -> x, w  # nodes, weights
+    gausshermite(n::Integer; probabilist = false) -> x, w  # nodes, weights
```
Comment (hyrodium) [inline src/gausshermite.jl:2]:
```suggestion
    gausshermite(n::Integer; normalize = false) -> x, w  # nodes, weights
```
Maybe, some other name of the keyword argument would be better?

Comment (NittanyLion) [reply src/gausshermite.jl]: I've made your suggested changes and added tests.

**Review by hyrodium — CHANGES_REQUESTED** [review]
Thank you for the updates! Could you remove `test/_test.todo`?

**Comment by NittanyLion** [comment]
> Thank you for the updates! Could you remove `test/_test.todo`?

done

**Review by hyrodium — APPROVED** [review]
LGTM! Thank you for the contribution!

**Pull request merged**